src/cmd/root.go

```diff
@@ -209,28 +174,27 @@ func init() {
 		return
 	}
 
-	v := getViper()
+	vpr := getViper()
 
 	// Logs
-	rootCmd.PersistentFlags().StringVarP(&LogLevelCLI, "log-level", "l", v.GetString(VLogLevel), lang.RootCmdFlagLogLevel)
-	rootCmd.PersistentFlags().StringVar(&LogFormat, "log-format", v.GetString(VLogFormat), "[beta] Select a logging format. Defaults to 'console'. Valid options are: 'console', 'json', 'dev', 'legacy'. The legacy option will be removed in a coming release")
-	rootCmd.PersistentFlags().BoolVar(&SkipLogFile, "no-log-file", v.GetBool(VNoLogFile), lang.RootCmdFlagSkipLogFile)
-	rootCmd.PersistentFlags().BoolVar(&message.NoProgress, "no-progress", v.GetBool(VNoProgress), lang.RootCmdFlagNoProgress)
```

Comment:
Would it make sense to keep this flag around for a bit and mark it as deprecated? Users probably have this flag in their pipeline, so their commands will fail when it's gone. I'm okay with it either way, but wanted to call this out.